tests/testthat/test-util-checkSpec.R

```diff
@@ -218,3 +220,32 @@ test_that("proper message appears when all data frames require `_all` columns",
     " No required columns specified"
   )
 })
+
+test_that("All dates are handles correctly", {
```

Comment:
was this supposed to be a date based test?

Reply:
oh whoops- i started writing a whole new test, but then ended up just testing dates in the "type" test above by including the  `SnapshotDate` column. I'll delete this now.